adventure/Kobold Press; Warlock Lair 50 The Light of Memoria.json

```diff
@@ -0,0 +1,320 @@
+				"Attuning to an amulet of memory removes any prior memories stored in it."
+			]
+		}
+	],
+	"monster": [
+		{
+			"name": "Spectral Memory",
+			"source": "WL50",
+			"page": 12,
+			"size": [
+				"M"
+			],
+			"type": "undead",
+			"alignment": [
+				"N"
+			],
+			"ac": [
+				12
+			],
+			"hp": {
+				"average": 22,
+				"formula": "5d8"
+			},
+			"speed": {
+				"walk": 0,
+				"fly": {
+					"number": 50,
+					"condition": "(hover)"
+				},
+				"canHover": true
+			},
+			"str": 1,
+			"dex": 14,
+			"con": 11,
+			"int": 15,
+			"wis": 12,
+			"cha": 10,
+			"senses": [
+				"darkvision 60 ft."
+			],
+			"passive": 11,
+			"resist": [
+				"acid",
+				"cold",
+				"fire",
+				"lightning",
+				"thunder",
+				{
+					"resist": [
+						"bludgeoning",
+						"piercing",
+						"slashing"
+					],
+					"note": "from nonmagical attacks",
+					"cond": true
+				}
+			],
+			"immune": [
+				"necrotic",
+				"poison"
+			],
+			"conditionImmune": [
+				"charmed",
+				"exhaustion",
+				"grappled",
+				"paralyzed",
+				"petrified",
+				"poisoned",
+				"prone",
+				"restrained",
+				"unconscious"
+			],
+			"languages": [
+				"Common",
+				"Elvish"
+			],
+			"cr": "1",
+			"trait": [
+				{
+					"name": "Incorporeal Movement",
+					"entries": [
+						"The spectral memory can move through other creatures and objects as if they were {@quickref difficult terrain|PHB|3}. It takes 5 ({@damage 1d10}) force damage if it ends its turn inside an object."
+					]
+				},
+				{
+					"name": "Light Sensitivity",
+					"entries": [
+						"While in bright light, the spectral memory has disadvantage on attack rolls, as well as on Wisdom ({@skill Perception}) checks that rely on sight."
+					]
+				},
+				{
+					"name": "Remembered Expertise",
+					"entries": [
+						"If the spectral memory makes an attack roll while at least one creature within 30 feet of it is missing a memory from Memory Drain, the spectral memory can roll a {@dice d4} and add the number rolled to the attack roll."
+					]
+				}
+			],
+			"action": [
+				{
+					"name": "Memory Drain",
+					"entries": [
+						"{@atk ms} {@hit 4} to hit, reach 5 ft., one creature. {@h}7 ({@damage 2d6}) necrotic damage. The target must succeed on a {@dc 10} Wisdom saving throw or lose a cherished memory. This loss lasts until the target finishes a short or long rest. Whenever a creature that is missing a memory makes an ability check or attack roll, it must roll a {@dice d4} and subtract the number rolled from the ability check or attack roll."
+					]
+				}
+			],
+			"traitTags": [
+				"Incorporeal Movement",
+				"Light Sensitivity"
+			],
+			"senseTags": [
+				"D"
+			],
+			"languageTags": [
+				"C",
+				"E"
+			],
+			"damageTags": [
+				"N",
+				"O"
+			],
+			"savingThrowForced": [
+				"wisdom"
+			],
+			"fluff": {
+				"_monsterFluff": {
+					"name": "Spectral Memory",
+					"source": "WL50"
+				}
+			}
```

Comment:
```suggestion
			},
			"hasFluff": true
```

Reply:
🤡

Reply:
dw i'm running my script again and there's a lot of other files needing it 😏

Reply:
https://github.com/TheGiddyLimit/homebrew/pull/1978/commits/64a39607fb8a8f69b6f70dff40943b9731b816c6
Resolved